docs/data/sdks/sdk-architecture.md

```diff
@@ -242,6 +242,24 @@ This a list of shared interfaces of latest SDKs.
 - `remove(Plugin)`
 - `shutdown()`
 
+## Identity
+
+Amplitude uses a combination of three different methods to identify your users: **device IDs**, **Amplitude ID**, and **user IDs**. Amplitude ID is automatically creates by Amplitude once a unique user is identified. You can assign a unique user ID for individual users, for example email addresses.
```

Comment:
I think we should list these in the order in which they are described. I would start with Device ID, then user ID, then Amplitude ID (since it is based on the other two).
```suggestion
Amplitude uses a combination of three different methods to identify your users: **device ID**, **user ID**, and **Amplitude ID**. 

Device ID is a unique ID for a given device. Since our client SDKs run on a single device they generate a device ID automatically. However server-side SDKs handle request for many devices, so a device ID must be provided manually for each event.

User ID ....  You can assign a unique user ID for individual users, for example email addresses.

Amplitude ID is automatically creates by Amplitude once a unique user is identified. Amplitude ID can include multiple device IDs and user IDs that are all associated with a single user.
```

Reply:
Fixed 

> User ID is an internal unique identifier configured by you. You can assign a unique user ID for individual users, for example email addresses.